src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -483,6 +483,10 @@ for (var type in topLevelEventsToDispatchConfig) {
 var ON_CLICK_KEY = keyOf({onClick: null});
 var onClickListeners = {};
 
+function getDictionaryKey(inst) {
+  return '.' + inst._rootNodeID;
+}
+
```

Comment:
Guys, seems pretty reasonable for me now. @gaearon Thanks!

Also it seems that `getDictionaryKey` should have a good explanation, comment above. V8's being developed so this could be irrelevant eventually.